restcomm/configuration/config-scripts/as7-config-scripts/restcomm/advanced.conf

```diff
@@ -97,6 +97,7 @@ RVD_UNDEPLOY='false'
 RVD_URL='' # override if RVD is not under the same domain with restcomm. Use a base url like http://myrvd.restcomm.com . No path included. This affects restcomm.xml/<rcmlserver/>
 RVD_VIDEO_SUPPORT='false'  # Toggle RVD Video RCML generation and UI
 RVD_MAX_MEDIA_FILE_SIZE=4194304 # Limit media file size when uploading. For video should probably be greater
+RVD_HTTP_TIMEOUT=''  # Timeout for http connections in rvd http client. It mainly affects connections to restcomm (create calls, fetch account etc.). It does not affect ES.
```

Comment:
do we need default value?. empty is using hardcoded? can you add comment with expected time units(secs,millis....)?

Reply:
@jaimecasero , is empty the setting (if exists) will be removed from rvd.xml and the default value in the rvd binary will be used.

Reply:
Let  me add the time units (ms)